screens/Home/HomeCard.tsx

```diff
@@ -242,11 +215,11 @@ export const HomeCard = (props: Props) => {
                     </TouchableOpacity>
 
                     <TouchableOpacity>
-                      <View style={globalStyles.sliderItem}>
-                        <View style={globalStyles.titleCard_Container}>
-                          <View style={globalStyles.boxArt_size}>
-                            <View style={globalStyles.rank_1_container}>
-                              <Rank_8 />
+                      <View style={homeStyles.sliderItem}>
+                        <View style={homeStyles.titleCard_Container}>
```

Comment:
we need to be consistent with casing. Javscript uses cammelCase.

so `homeStyles.titleCard_Container` should be `homeStyles.titleCardContainer`